### ISSUE TITLE
Bug: mistake in "Customer preview" (/compta/recap-compta.php) calculations ?

### ISSUE THREAD

**Issue by sbschumacher** [Dolibarr/dolibarr#5329]
# Bug

There seems to be a mistake in the calculation of the balance because credit notes and deposit invoices are also included.

Example:
![recap-compta](https://cloud.githubusercontent.com/assets/14094327/15890566/882eb28e-2d70-11e6-838f-defc20942702.jpg)

Btw: The amount stated in the line "Current outstanding bill" status at /comm/card.php?socid=xxx is correct like it should be in the image above.
## Environment
- **Version**: 9.3 - 9.3.2
- **OS**: Linux
- **Web server**: Apache
- **PHP**: 5.6.22
- **Database**: MySQL 5.1.73-log
- **URL(s)**: /compta/recap-compta.php

**Comment by atm-maxime** [comment]
Hello @sbschumacher ,

In which Dolibarr version are you ? This page has been reviewed in the 3.9 version.
It's possible deposit are not properly handled regarding to the option you are in, but I prefer you check your version first.

Thanks

**Comment by sbschumacher** [comment]
Hi @atm-maxime,

I am sorry for the confusion with "9.3 - 9.3.2"... currently I am using **3.9.2**.

The SQL-query in /compta/recap-compta.php from line 85 to 92 does not exclude deposits and credit notes:
![sql](https://cloud.githubusercontent.com/assets/14094327/15985139/41fe256c-2fe4-11e6-925a-5e7981f4d7ef.jpg)

**Comment by atm-maxime** [comment]
Hi @sbschumacher ,

I think that this is normal to have credit note in this balance. When you record a credit note, the customer account is credited with the amount and will be debited when refunded or used on another invoice. In your example, there's a credit note for 70, and this will be balanced by a refund of 70 in the debit, or by an invoice that will use this credit note to have 0 remaining to pay.

Regarding the deposit, unless you are working with the option FACTURE_DEPOSITS_ARE_JUST_PAYMENTS, in which case I agree that only the payment of the deposit should appear and not the deposit itself, a deposit is just like a regular invoice, so should be there too for me.

In short, for me any invoice (except deposits if just payments) should appear, and any payment too. The calculation of all of it make the customer account value.

As I am not an expert at all, could please someone else have a look and tell us ?

**Comment by simnandez** [comment]
No answer

**Comment by BebZ** [comment]
Hey, it has been closed and / or reviewed ?
Your point of vieuw is right : In short, for me any invoice (except deposits if just payments) should appear, and any payment too. The calculation of all of it make the customer account value.

We are trying to remake the recap-compta with @Darkjeff 
is someone working on it, or a module to integrate in core 
or are there any other useful informations to know ?
